tests/popup-test.js

```diff
@@ -13,10 +13,71 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+require("../readiness-tool/popup");
 
-describe('test', function() {
+var chai = require('chai');
 
-   it('should pass', function() {
-     expect(1).to.equal(1);
-   });
- });
+var vendors = require("../readiness-tool/vendors");
+
+describe('isSupported(key)', function () {
+
+  it('should return true when the key given is in the supported list', function () {
+    expect(self.popups.isSupported('Adhese')).to.be.true;
```

Comment:
The word 'export' being in the chrome extension stops it from running successfully.